src/routes/settings/Interface.svelte

```diff
@@ -31,7 +31,8 @@
 			{ value: 'en', label: 'English' },
 			{ value: 'es', label: 'Español' },
 			{ value: 'ja', label: '日本語' },
-			{ value: 'tr', label: 'Türkçe' }
+			{ value: 'tr', label: 'Türkçe' },
+			{ value: 'pt-br', label: 'Português do Brasil' }
```

Comment:
Would you mind renaming the label `Português (Brasil)` so it's a little shorter?